client_test.go

```diff
@@ -1681,6 +1681,23 @@ func TestGocloak_Logout(t *testing.T) {
 	require.NoError(t, err, "Logout failed")
 }
 
+func TestGocloak_LogoutAllSessions(t *testing.T) {
+	t.Parallel()
+	cfg := GetConfig(t)
+	client := NewClientWithDebug(t)
+	token := GetAdminToken(t, client)
+
+	tearDown, userID := CreateUser(t, client)
+	defer tearDown()
+
+	err := client.LogoutAllSessions(
+		context.Background(),
+		cfg.GoCloak.Realm,
+		userID,
+		token.AccessToken)
```

Comment:
I'm getting `"error": "HTTP 401 Unauthorized"` because you are trying to log out a created user using admin toke.
So, you need:
1. CreateUser (done)
2. Obtain a token for that user
3. Log out that suer (partially done, jus use a token from step 2)

Reply:
My bad. 
I dont think that's the issue. You'd get 401 Unauthorized if you use the Admin token to logout since it uses /auth/admin 
I didnt change the order at gocloak.go. I'll push an update

Reply:
It's using the userID as the token and the token as the userID, that's why it fails 

`OST  /auth/admin/realms/9db3f6d3-a7ab-4299-a0ae-41e4a0d981d8/users/eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJtNGFqY0EtdTBweElSQkVVa3hGN1RCVU1DcnlVVkowQnZRU2xndDM0U0NFIn0.eyJleHAiOjE1OTM1MjQwOTEsImlhdCI6MTU5MzUyNDAzMSwianRpIjoiMzVlNjQ0NWMtZjhhMC00NzdmLTg2ZjYtZGJiNjc2M2U4OGJhIiwiaXNzIjoiaHR0cDovL2xvY2FsaG9zdDo4MDgwL2F1dGgvcmVhbG1zL21hc3RlciIsInN1YiI6ImZiYTQ0ZGIzLTFlYWQtNGM1ZC1hZjQ0LTk0NjRlMDU1NjMzMyIsInR5cCI6IkJlYXJlciIsImF6cCI6ImFkbWluLWNsaSIsInNlc3Npb25fc3RhdGUiOiJhYjU2ZjIxOS1mNjE4LTQyNTAtYjQ1ZS1lZmI3MDE5MmIwYjAiLCJhY3IiOiIxIiwic2NvcGUiOiJwcm9maWxlIGVtYWlsIiwiZW1haWxfdmVyaWZpZWQiOmZhbHNlLCJwcmVmZXJyZWRfdXNlcm5hbWUiOiJhZG1pbiJ9.CoxpFtpFzrXUVKDAFWPkxL73R4BItdQ2SfPAW8BuSBOQ3RvuLcYvE2WgKLY5BqQWLP7LimwVEm6NLEy0BHKZAicMqkkcVwKPQCPqAFQSZRSdyGVD5RQr2tIi40nQ7f82qw9HzSPA3OwvrAhONH91u6S3fK644W4VTkE4mXwBNWziYK-fRvSZ1J-TOQmDRZe_hlrdr7OC9pyA6tEDVMZipS8fRw6p0Xq6lizag4ciI0xGvzhrgZ9y4GjWq37bUoU2y0zRG8eU_-51K1psWABhdLPMpUfCxx1F5_KJaSxCuO4L_Bp9z2caaEI7nOxsqsxnzPr-eIhJCYgfp-5TWPfL8A/logout `

Reply:
oh, I see, thank you. Actually it means that we can log out any users/sessions using admin token